config/honeybadger.yml

```
@@ -0,0 +1,2 @@
+---
+api_key: 'ea84bd0a'
```

I do not think we want to add the api_key in clear text like this.
```suggestion
api_key: <%= ENV.fetch('HONEYBADGER_API_KEY', '') %>
```